scheduler/test/cook/test/mesos/util.clj

```diff
@@ -59,6 +61,34 @@
             (concat (take 100 (filter odd? (range)))
                     (drop 100 (filter even? (range)))))))
 
+(deftest test-task-run-time
+  (let [uri "datomic:mem://test-task-run-time"
+        conn (restore-fresh-database! uri)]
+    (testing "task complete"
+      (let [expected-run-time (t/hours 3)
+            start (t/ago expected-run-time)
+            end (t/now)
```

Comment:
Some milliseconds may occur between the calls to t/ago and t/now, which can throw off this test randomly